specification/common-types/resource-management/v4/managedidentitywithdelegation.json

```diff
@@ -0,0 +1,54 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "4.0",
+    "title": "Common types"
+  },
+  "paths": {},
+  "definitions": {
+    "ManagedServiceIdentityWithDelegation": {
+      "description": "Managed service identity (system assigned and/or user assigned identities and/or delegated identities) - internal use only.",
```

Comment:
These properties are only internally available to other RPs, but not to end users. @msyyc @ArcturusZhang @qiaozha Is it possible to avoid exposing these properties in SDKs? Is the description enough?

Reply:
and also hide it in documentation.
i see it's hidden in https://docs.microsoft.com/en-us/rest/api/sql/2021-11-01-preview/databases/create-or-update?tabs=HTTP#databaseidentity somehow.

Reply:
They're not defining `delegatedResources` in their swagger:
https://github.com/Azure/azure-rest-api-specs/blob/8c2258114e565bb041dcb25a761acb965a4870e7/specification/sql/resource-manager/Microsoft.Sql/preview/2021-11-01-preview/Databases.json#L821-L851

Reply:
sorry, i was looking on master rather than main.

so question remains the same - Is it possible to avoid exposing these properties in SDKs and documentation?